wikimedia/wikipedia-ios
# Apple Pay - Update transaction fee as a percentage of the donation amount

**Phabricator:** https://phabricator.wikimedia.org/T365306

### Notes
This PR builds off of https://github.com/wikimedia/wikipedia-ios/pull/4885 and updates the transaction fee to be a percentage of the donation amount, rather than using a static default.

### Test Steps
1. Go to Apple Pay form and play with the amount presents. Confirm the transaction fee updates to be .04% of the donation amount. Change the textfield to a custom donation amount. Confirm the transaction reflects accordingly.
2. Change the currency type by change the device's iOS Settings > General > Language and Region > Region. Confirm the transaction fees work as expected.
